DEBUG - 2025-06-13 16:37:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:38:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:38:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-13 16:38:18 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'projeto', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(609): CodeIgniter\Model->doFind(false, NULL)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Login.php(29): CodeIgniter\BaseModel->find()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Login->logar()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter4\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(609): CodeIgniter\Model->doFind(false, NULL)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Login.php(29): CodeIgniter\BaseModel->find()
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Login->logar()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2025-06-13 16:38:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
[Method: POST, Route: login/logar]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuarios_cpf` = :usuarios_cpf:
OR `usuarios_email` = :usuarios_email:
AND `usuarios_senha` = :usuarios_senha:', [...], false)
 3 SYSTEMPATH\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(609): CodeIgniter\Model->doFind(false, null)
 5 APPPATH\Controllers\Login.php(29): CodeIgniter\BaseModel->find()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->logar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 16:38:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 16:39:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-13 16:39:59 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `estoque...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `estoque...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `estoque...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `estoque...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Estoques.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Estoques->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Estoques))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-13 16:39:59 --> CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
[Method: GET, Route: estoques]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Estoques.php(13): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Estoques->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Estoques))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 16:39:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Estoques.php(13): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Estoques->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Estoques))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 16:39:59 --> [Caused by] mysqli_sql_exception: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `estoque`.*, `produtos`.`produtos_nome`
FROM `estoque`
JOIN `produtos` ON `produtos`.`produtos_id` = `estoque`.`produto_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Estoques.php(13): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Estoques->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Estoques))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 16:42:59 --> ErrorException: Undefined variable $form
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 4.
 1 APPPATH\Views\pedidos\form.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $form', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 4)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(22): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 16:43:09 --> ErrorException: Undefined variable $form
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 4.
 1 APPPATH\Views\pedidos\form.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $form', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 4)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(22): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 16:52:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-13 16:54:53 --> Error: Call to undefined function App\Controllers\msg()
[Method: POST, Route: estoques/update/1]
in APPPATH\Controllers\Estoques.php on line 58.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Estoques->update('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Estoques))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 17:03:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:03:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:03:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-13 17:08:54 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/1]
in APPPATH\Controllers\Vendas.php on line 80.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:09:15 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/1]
in APPPATH\Controllers\Vendas.php on line 80.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:12:46 --> ErrorException: Attempt to read property "usuario_id" on null
[Method: GET, Route: vendas/show/1]
in APPPATH\Controllers\Vendas.php on line 81.
 1 APPPATH\Controllers\Vendas.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuario_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 81)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('1')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:15:09 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/1]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('1')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 17:15:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 17:15:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-13 17:15:46 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 17:15:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-13 17:16:29 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:16:55 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:18:25 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 95.
 1 APPPATH\Controllers\Vendas.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 95)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:18:29 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 95.
 1 APPPATH\Controllers\Vendas.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 95)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:31:35 --> ErrorException: Undefined property: stdClass::$usuarios_id
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:31:40 --> ErrorException: Undefined property: stdClass::$usuarios_id
[Method: GET, Route: vendas/show/12]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('12')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 17:31:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-13 17:31:48 --> ErrorException: Undefined property: stdClass::$usuarios_id
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-13 17:33:10 --> mysqli_sql_exception: Unknown column 'usuarios.nome' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Vendas.php(81): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-13 17:33:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.nome' in 'field list'
[Method: GET, Route: vendas/show/13]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = :vendas.venda_id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Vendas.php(81): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:33:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuarios.nome' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = \'13\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = \'13\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = :vendas.venda_id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Vendas.php(81): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:33:10 --> [Caused by] mysqli_sql_exception: Unknown column 'usuarios.nome' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = \'13\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = \'13\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = \'13\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `usuarios`.`usuarios_id`, `usuarios`.`nome`, `usuarios`.`usuarios_sobrenome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`cliente_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `clientes`.`usuario_id`
WHERE `vendas`.`venda_id` = :vendas.venda_id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Vendas.php(81): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:33:16 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 103.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:34:44 --> ErrorException: Attempt to read property "produtos_produtos_id" on array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 102.
 1 APPPATH\Controllers\Vendas.php(102): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "produtos_produtos_id" on array', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 102)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:36:56 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:36:58 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:37:08 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:37:10 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:37:11 --> ErrorException: Attempt to read property "usuarios_id" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 91.
 1 APPPATH\Controllers\Vendas.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Vendas.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:41:30 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Views\vendas\show.php on line 4.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('vendas/show', [], true)
 4 APPPATH\Controllers\Vendas.php(111): view('vendas/show', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:41:36 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Views\vendas\show.php on line 4.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('vendas/show', [], true)
 4 APPPATH\Controllers\Vendas.php(111): view('vendas/show', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:18 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:20 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:20 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:21 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:22 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:43:49 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:44:05 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:45:33 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:45:34 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:45:35 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:45:35 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:47:14 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:47:16 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:47:19 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:48:05 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:48:06 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:48:22 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:48:23 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:51:46 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:53:12 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:53:15 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: vendas/show/13]
in APPPATH\Controllers\Vendas.php on line 105.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:55:09 --> ErrorException: Attempt to read property "usuarios_nome" on null
[Method: GET, Route: vendas/show/13]
in APPPATH\Views\vendas\show.php on line 5.
 1 APPPATH\Views\vendas\show.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "usuarios_nome" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php', 5)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('vendas/show', [], true)
 5 APPPATH\Controllers\Vendas.php(111): view('vendas/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:56:00 --> ErrorException: Undefined variable $usuarios
[Method: GET, Route: vendas/show/13]
in APPPATH\Views\vendas\show.php on line 5.
 1 APPPATH\Views\vendas\show.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usuarios', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php', 5)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('vendas/show', [], true)
 5 APPPATH\Controllers\Vendas.php(111): view('vendas/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-13 17:56:02 --> ErrorException: Undefined variable $usuarios
[Method: GET, Route: vendas/show/13]
in APPPATH\Views\vendas\show.php on line 5.
 1 APPPATH\Views\vendas\show.php(5): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usuarios', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php', 5)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\vendas\\show.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('vendas/show', [], true)
 5 APPPATH\Controllers\Vendas.php(111): view('vendas/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->show('13')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-13 18:19:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-13 20:49:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
